awx_collection/plugins/modules/tower_workflow_job_template_node.py

```diff
@@ -174,6 +174,54 @@
     organization: Default
     success_nodes:
       - my-first-node
+
+- name: Create workflow with 2 Job Templates and an approval node in between
+  block:
+  - name: Create a workflow job template
+    awx.awx.tower_workflow_job_template:
```

Comment:
Can you drop the FQCNs? It should be consistent with the examples above, ie just `tower_workflow_job_template:`

Reply:
done, thanks for the hint